shared/src/main/java/com/couchbase/lite/ReadOnlyDocument.java

```diff
@@ -192,19 +192,20 @@ else if (c == '-')
         return true;
     }
 
-    /* package */ boolean selectCommonAncestor(ReadOnlyDocument doc1, ReadOnlyDocument doc2) {
+    boolean selectCommonAncestor(ReadOnlyDocument doc1, ReadOnlyDocument doc2)
+            throws CouchbaseLiteException {
```

Comment:
Is CouchbaseLiteException thrown from anything in this method? 
Or it should return void but throw CouchbaseLiteException instead?